src/components/calendar/EventModal.js

```diff
@@ -0,0 +1,187 @@
+import React, {useState} from 'react';
+import {Button, Modal, Image, Card} from 'react-bootstrap';
+import './EventModal.css';
+
+function checkIfUndefined(attribute) {
+
+    if(typeof(attribute) !== "undefined") {
+        return attribute;
+    }
+
+    return '';
+}
+
+function formatAMPM(date) {
```

Comment:
This is good, but for the future, you can consider making a utils class. This way, you can use these functions multiple times and it also helps with your code clarity.